datasets/md/terror_sanctions/md_terror_sanctions.yml

```diff
@@ -54,6 +54,15 @@ data:
   format: HTML
 dates:
   formats: ["%d.%m.%Y", "%d %b. %Y", "%b. %Y", "%b.%Y"]
+tags:
+  - issuer.md                         # Published by the Republic of Moldova
+  - list.sanction                     # A sanctions list targeting specific persons, entities, and groups
+  - target.md                         # Focused on Moldova and relevant legal context
+  - juris.md                          # Relevant to the legal jurisdiction of Moldova
+  - risk.terror                       # Focus on individuals/groups involved in terrorism
+  - issue.proliferation               # Related to the proliferation of weapons of mass destruction
```

Comment:
it feels a bit like list.sanction, risk.terror and issue.proliferation are all kind of circling the same ideas.

Are there cases where someone needs as much detail as proliferation as opposed to `issue.bombings` or whatever?